pkg/nfd-master/nfd-master.go

```diff
@@ -288,7 +311,7 @@ func (s *labelerServer) SetLabels(c context.Context, r *pb.SetLabelsRequest) (*p
 
 // advertiseFeatureLabels advertises the feature labels to a Kubernetes node
 // via the API server.
-func updateNodeFeatures(helper apihelper.APIHelpers, nodeName string, labels Labels, annotations Annotations) error {
+func updateNodeFeatures(helper apihelper.APIHelpers, nodeName string, labels Labels, annotations Annotations, resourceLabels ResourceLabels) error {
```

Comment:
I'd change the last argument to be `ExtendedResources` that will be advertised as is. This would make this function more general and future-proof